content/Projects/Observability/RHOBS/README.md

```diff
@@ -1,5 +1,63 @@
 # Red Hat Observability Service
 
-Status: Open to internal Red Hat Teams Only.
+*Previous Documents:*
 
-TBD
+* [Initial Design for RHOBS (Internal Content)](https://docs.google.com/document/d/1cSz_ZbS35mk8Op92xhB9ijW1ivOtJuD1uAzPiBdSUqs/edit)
+* [Initial FAQ (Internal Content)](https://docs.google.com/document/d/1_xnJBS3v7n4m229L3tqCqBXzZy55yu6dxCJY-vh_Egs/edit)
+
+## What
+
+Red Hat Observability Service (RHOBS) is a managed, centralized, multi-tenancy, scalable backend for the observability data. Functionally it is an [Observatorium](../observatorium.md) as A Service (SaaS) installation. RHOBS is meant to allowing ingesting, storing and using (visualisations, import, alerting, correlation) observability signals like metrics, logging and tracing is using well established APIs.
+
+This document outlines the basic overview of the RHOBS service. If you want to learn about RHOBS architecture, look for [Observatorium](https://observatorium.io) default deployment and it's design.
```

Comment:
"Outlining an overview of X" is pretty abstract, i think we should either "provide an overview of X" or "outline X"